go.mod

```diff
@@ -49,8 +49,10 @@ require (
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/openshift/assisted-service/api v0.0.0
 	github.com/openshift/assisted-service/models v0.0.0
+	github.com/stmcginnis/gofish v0.16.1
 	github.com/stretchr/testify v1.9.0
 	github.com/vmware-tanzu/velero v1.12.1
+	golang.org/x/crypto v0.19.0
```

Comment:
https://github.com/golang/crypto/releases/tag/v0.22.0

Reply:
This direct dependency has been removed as SSH access will be implemented in separate PR.